language/vm/src/file_format.rs

```diff
@@ -155,8 +163,10 @@ pub type MemberCount = u16;
 /// the instruction stream.
 pub type CodeOffset = u16;
 
-/// The pool of identifiers and string literals.
+/// The pool of identifiers.
 pub type StringPool = Vec<String>;
```

Comment:
Will `IdentifierPool` be a better name for this pool?

Reply:
yep, coming in the next series of diffs :)

Reply:
I'll add a comment.